crates/sql-backend-handler/src/sql_migrations.rs

```diff
@@ -1112,6 +1119,55 @@ async fn migrate_to_v10(transaction: DatabaseTransaction) -> Result<DatabaseTran
     Ok(transaction)
 }
 
+async fn migrate_to_v11(transaction: DatabaseTransaction) -> Result<DatabaseTransaction, DbErr> {
+    let builder = transaction.get_database_backend();
+
+    // Check if there's a custom attribute named "login_enabled"
+    #[derive(FromQueryResult)]
+    struct AttributeCheck {
+        count: i64,
+    }
+
+    let existing_attribute = AttributeCheck::find_by_statement(
+        builder.build(
+            Query::select()
+                .expr(Func::count(Expr::col(
+                    UserAttributeSchema::UserAttributeSchemaName,
+                )))
+                .from(UserAttributeSchema::Table)
+                .and_where(
+                    Expr::col(UserAttributeSchema::UserAttributeSchemaName).eq("login_enabled"),
+                ),
+        ),
+    )
+    .one(&transaction)
+    .await?;
+
+    if let Some(check) = existing_attribute
+        && check.count > 0
+    {
+        warn!(
```

Comment:
I think this should be a hard error. You can't upgrade until you rename/delete this attribute.